.github/workflows/haskell.yml

```diff
@@ -137,8 +137,8 @@ jobs:
           cache-cabal-store-v1-${{ runner.os }}-${{ matrix.ghc }}-${{ hashFiles('dependencies.txt') }}
           cache-cabal-store-v1-${{ runner.os }}-${{ matrix.ghc }}-
 
-    - name: Build dependencies
-      run: cabal build all --builddir="$CABAL_BUILDDIR" --only-dependencies
+          #    - name: Build dependencies
```

Comment:
This was commented out because it raises compile errors. We can simply build everything.

Reply:
Compile errors in what? Dependencies?

Reply:
Yes. See https://input-output-rnd.slack.com/archives/CAP8NM7N0/p1616416227016800